vlib/v/fmt/tests/array_init_keep.vv

```diff
@@ -0,0 +1,9 @@
+fn main() {
+	arr := [
+		// test 0
+		1,
+		// test 1
+		2,
```

Comment:
There should not be a comma here? The last element in arrays should not be followed by a comma. You've done this in all of your tests.

Reply:
See the thread above with Spytheman : https://github.com/vlang/v/pull/5851#discussion_r456303454

Reply:
Oh, right. My bad. Good job :slightly_smiling_face: